cpp/src/io/shp/polygon_shapefile_reader.cpp

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#include <ogrsf_frmts.h>
 #include <cuspatial/error.hpp>
+#include <ogrsf_frmts.h>
```

Comment:
Separate headers from different libraries by a blank line.

```suggestion
#include <cuspatial/error.hpp>

#include <ogrsf_frmts.h>
```